nextjs-logs-manager/components/DropdownNavBarLink.tsx

```
@@ -0,0 +1,53 @@
+      // Redirigir a la página de login después de hacer logout
+      localStorage.removeItem("token");
+      localStorage.removeItem("user");
+      localStorage.removeItem("valid");
+    } else {
+      // Show error message to user
+      alert("Failed to logout. Please try again.");
+    }
+  };
+
+  return (
+    <Dropdown hidden={isLoggedIn ? false : true} label={user}>
```

Biome (1.9.4)

[error] Unnecessary use of boolean literals in conditional expression.

Simplify your code by directly assigning the result without using a ternary operator.
If your goal is negation, you may use the logical NOT (!) or double NOT (!!) operator for clearer and concise code.
 Check for more details about NOT operator.
Unsafe fix: Remove the conditional expression with


(lint/complexity/noUselessTernary)